docs/blog/2022-07-16_minio-test/2022-07-16_minio-test.md

```diff
@@ -1,8 +1,338 @@
 ﻿---
 slug: minio
-title: HwameiStor Supports Minio
-authors: [Simon]
+title: HwameiStor Supports MinIO
+authors: [Simon, Michelle]
 tags: [Test]
 ---
 
-# HwameiStor supports Minio
\ No newline at end of file
+# HwameiStor Supports MinIO
+
+## MinIO Introduction
+
+MinIO is a high performance object storage solution with native support for Kubernetes deployment. It can provide distributed, S3-compatible, and multi-cloud storage service in public cloud, private cloud, and edge computing scenarios. MinIO is software-defined and open source under GNU AGPL v3. It can also run well on x86 and other standard hardware.

```

Comment:
```suggestion
MinIO is a high performance object storage solution with native support for Kubernetes. It can provide distributed, S3-compatible, and multi-cloud storage service in public cloud, private cloud, and edge computing scenarios. MinIO is software-defined product, open sourced with the GNU Affero General Public License. It can also run well on x86 and other standard hardware.
```